src/stories/Library/event-list-item/event-list-item.scss

```
@@ -165,13 +231,19 @@ $_desktop-content-rows: 5;
     grid-row: 3 / span 3;
     justify-content: unset;
   }
+
+  .event-list-item-stacked {
+    grid-template-areas: ". contentStacked";
```

```suggestion
    grid-template-areas: ". content";
```

Again just suggestion: 

If everything is part of 'stacked' anyway, i would just call the areas for content, time, arrow, etc. If you prefer this that's also fine, up to you.

---

Do you know if it makes any difference to have several aria names with the same name? does it only look at the classes that are children of the grid?